kbrownell/drupal-caravan
# Run DrupalVM playbook with a caravan playbook

# READY FOR REVIEW

# Summary
- The previous version of drupal-caravan overwrote config files in the web directory.  This pull request removes unnecessary config files, ie. project.yml, runs behat from config files within the caravan directory, and sets up the skeleton for running drupalvm as a playbook within a master, caravan playbook.  This allows us to keep the caravan config files within the caravan directory, while still building off of drupalvm.

# Needed By (Date)
- The sooner the better, so I can keep moving forward.

# Urgency
- This is blocking further progress on cleaning up drupal-caravan.

# Steps to Test

1. In the se3_blt repository, change the drupal-caravan version to `run-with-ansible.`
2. Run `composer update` and make sure you see an update listed for drupal-caravan.
3. Run `vendor/su-sws/drupal-caravan/setup-caravan.sh`
4. See if it builds a container named `earth.`

# Affected Projects or Products
- Earth and our ability to set up local and CI test environments quickly and easily.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/EARTH-260
